packages/@webex/plugin-meetings/src/rtcMetrics/index.ts

```diff
@@ -47,9 +49,7 @@ export default class RtcMetrics {
     this.meetingId = meetingId;
     this.webex = webex;
     this.correlationId = correlationId;
-    this.setNewConnectionId();
-    // Send the first set of metrics at 5 seconds in the case of a user leaving the call shortly after joining.
-    setTimeout(this.sendMetricsInQueue.bind(this), 5 * 1000);
+    this.resetConnection();
```

Comment:
I've renamed `setNewConnectionId()` to `resetConnection()`, because we need to reset the `initialMetricsSent` flag when we have a new connection, so that we upload the stats for the new connection as soon as they appear